packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -1,5 +1,6 @@
 import { FormattedMessage, intlShape } from 'react-intl';
 import Button from 'terra-button';
+import moment from 'moment';
```

Comment:
Where is this being used?

Reply:
This is being used by `startOf` [here](https://github.com/cerner/terra-framework/pull/994/files#diff-f0779fb2b55a5a8b2bef87c374f19eceR396).

@saket2403 Would you be able to use `getStartOfMonth` from [date_util](https://github.com/cerner/terra-framework/blob/ed759f73d487259b057b34818cc3b352b1098e3f/packages/terra-date-picker/src/react-datepicker/date_utils.js#L148)?

Reply:
Changed 76967e7